bytecode/__init__.py

```diff
@@ -1,22 +1,23 @@
+from __future__ import print_function
+
 __version__ = '0.5'
 
 __all__ = ['Label', 'Instr', 'SetLineno', 'Bytecode',
            'ConcreteInstr', 'ConcreteBytecode',
            'ControlFlowGraph']
 
-from bytecode.instr import (UNSET, Label, SetLineno, Instr, CellVar, FreeVar,   # noqa
-                            Compare)
-from bytecode.bytecode import BaseBytecode, _InstrList, Bytecode   # noqa
-from bytecode.concrete import (ConcreteInstr, ConcreteBytecode,   # noqa
+from .instr import (UNSET, Label, SetLineno, Instr, CellVar, FreeVar,   # noqa
+                    Compare)
```

Comment:
I dislike such change. Can you try with "from __future__ import absolute_import"?